code/modules/paperwork/photocopier.dm

```diff
@@ -184,6 +184,15 @@
 			usr << "<span class='notice'>You take [photocopy] out of [src].</span>"
 			photocopy = null
 			updateUsrDialog()
+		else if(doccopy)
+			if(!istype(usr,/mob/living/silicon/ai)) // honk honk honk
+				doccopy.loc = usr.loc
+				usr.put_in_hands(doccopy)
+			else
+				doccopy.loc = src.loc
+			usr << "<span class='notice'>You take [doccopy] out of [src].</span>"
+			doccopy = null
+			updateUsrDialog()

```

Comment:
make a photocopier proc out of this, this bunch of lines is copypasted two times just above.